core/market/tests/test_cyclic_broadcasts.rs

```diff
@@ -17,14 +16,8 @@ use ya_market::testing::{
 async fn test_startup_offers_sharing() {
     let _ = env_logger::builder().try_init();
 
-    // Change expected time of sending broadcasts.
-    let mut config = Config::default();
-    config.discovery.mean_cyclic_bcast_interval = Duration::from_millis(100);
-    config.discovery.max_bcasted_offers = 7;
-
```

Comment:
This was for purpose, please don't remove here and in any tests that creates config.
If config isn't default, that means, that test depends on values that are set here. You shouldn't use default config, that's values can change at any point in time

Reply:
As you saw, I've created a dedicated config for **all** market suite tests (vide `create_market_config_for_test`).
It is crafted in a way, which does not require special overwrites within any particular test case.

Is that ok?

Reply:
I've also moved test helper code which is waiting for broadcasts near afromentioned config creation
See `fn assert_offers_broadcasted`

Reply:
> As you saw, I've created a dedicated config for **all market suite tests** (vide `create_market_config_for_test`).
> It is crafted in a way, which does not require special overwrites within any particular test case.
> 
> Is that ok?

Most of the tests can work with any value from this config. These were prepared specifically for the value that were set here. When you change this general config in future, you are not aware, that you are breaking these tests. So I prefer to set them explicite here

Reply:
We agreed on that during video call